sympy/physics/mechanics/tests/test_joint.py

```diff
@@ -207,3 +207,20 @@ def generate_body():
         (1 - sqrt(3)/3)*N.x + sqrt(6)*sin(theta + pi/4)/3*N.y - sqrt(6)*cos(theta + pi/4)/3*N.z
     assert C.masscenter.vel(N).express(N).simplify() == sqrt(6)*omega*cos(theta + pi/4)/3*N.y + sqrt(6)*omega*sin(theta + pi/4)/3*N.z
     assert C.masscenter.vel(N).angle_between(A.x) == pi/2
+
+    N, A, P, C = generate_body()
+    a, b, c = symbols('a b c')
+    m, n = symbols('m n')
+    PinJoint('J', P, C, parent_joint_pos=m*N.x, child_joint_pos=n*A.x, child_axis=a*A.x+b*A.y-c*A.z, parent_axis=a*N.x-b*N.y+c*N.z)
+    assert (a*N.x-b*N.y+c*N.z).angle_between(a*A.x-b*A.y+c*A.z).simplify() == 0 #Axis are aligned
+    assert (a*A.x-b*A.y+c*A.z).express(N).simplify() == sqrt(3)*N.x
+    assert A.ang_vel_in(N) == omega*N.x
+    assert A.ang_vel_in(N).express(A).simplify() == (omega*A.x + omega*A.y - omega*A.z)/sqrt(3)
+    assert A.ang_vel_in(N).magnitude() == omega
+    assert A.ang_vel_in(N).angle_between(a*A.x-b*A.y+c*A.z).simplify() == 0
+    assert C.masscenter.vel(N).simplify() == (omega * A.y + omega * A.z)/sqrt(3)
```

Comment:
The magnitude of the linear velocity should be n*omega, but n isn't present here.